cmake/mingw32-w64-cross-travis.toolchain.cmake

```diff
@@ -0,0 +1,53 @@
+#
+# Mingw-w64 cross compiler toolchain
+#
+# - The usual CMAKE variables will point to the cross compiler
+# - HOST_EXE_LINKER, HOST_C_COMPILER, HOST_EXE_LINKER_FLAGS,
+#   HOST_C_FLAGS point to a host compiler
+#
+
+set(MINGW_TRIPLET i686-w64-mingw32)
+# For x86_64 use
+#set(MINGW_TRIPLET x86_64-w64-mingw32)
+
+# The location of your toolchain sys-root
+set(MINGW_PREFIX_PATH /opt/mingw32/${MINGW_TRIPLET}/)
+# or sometimes like this
+#set(MINGW_PREFIX_PATH /usr/${MINGW_TRIPLET}/sys-root)
```

Comment:
I see.  We need to build msgpack via CMake, and it won't pick up the correct value.  We could just add it to the command line there, just like we're adding the toolchain file.  As I've said before though, I don't think all of this needs to get worked out before merging this.  Let's leave it the way it is for now, and perhaps a better solution will emerge down the road.